docs/page.css

```diff
@@ -40,11 +40,11 @@ h3 {
 	font-size: 16px;
 	font-weight: normal;
 
-	margin-top: 30px;
+	margin-top: 40px;
 }
 
 div {
-	/* padding-left: 30px; */
+    padding-left: 30px; 
```

Comment:
Oh - sorry! 

The manual pages seem to have a partly inconsistent CSS, and when testing the effects of my CSS changes on the manual pages I must have accidentially chosen those that looked good, since they use `<div class=”desc“>` (such as “Detecting WebGL”) or `<p>` (such as “Matrix transformations”).

My suggestion:

I would like to unify the “Manual” and “Constant” pages’ CSS by assigning div `<class=”desc”>` to all paragraphs that shall not be indented. That would be not much work because only a few pages are affected. In this way my CSS changes would have no negative side effects.

Would that be okay?